src/sagemaker/session.py

```diff
@@ -29,8 +29,8 @@
 import sagemaker.logs
 from sagemaker import vpc_utils
 
-# import s3_input for backward compatibility
-from sagemaker.inputs import s3_input  # noqa # pylint: disable=unused-import
+# import TrainingInput for backward compatibility
+from sagemaker.inputs import TrainingInput  # noqa # pylint: disable=unused-import
```

Comment:
what do you think of removing this since we're making a breaking change anyway?

Reply:
Yes, I think we should remove this as well.

I've updated the changes including the docstrings.